src/components/AvatarWithImagePicker.js

```diff
@@ -321,9 +308,21 @@ class AvatarWithImagePicker extends React.Component {
                                     <PopoverMenu
                                         isVisible={this.state.isMenuVisible}
                                         onClose={() => this.setState({isMenuVisible: false})}
-                                        onItemSelected={() => this.setState({isMenuVisible: false})}
-                                        menuItems={this.createMenuItems(openPicker)}
+                                        onItemSelected={(item, index) => {
+                                            this.setState({isMenuVisible: false});
+                                            // In order for the file picker to open dynamically, the click
+                                            // function must be called from within a event handler that was initiated
+                                            // by the user.
```

Comment:
Yeah Safari has some hard-to-please requirements to prevent what they think are "pop-ups" so most stuff has to be user-driven. @narefyev91 or @aimane-chnaif any additional thoughts on how we might get around this?